features/pubkeys/show.feature

```diff
@@ -12,7 +12,3 @@ Feature: Show public keys for a user
   Scenario: After adding a key, the key is shown
     When I run `conjur pubkeys show alice`
     And the output should match /^ssh-rsa .* laptop$/
-
-  Scenario: Public keys can be listed using cURL, without authentication
```

Comment:
Why was this test removed?

Reply:
https://github.com/cyberark/conjur-cli/pull/230/commits/672867b109f65f57fe73f55f4a4f840b4e461544

> Remove a feature that tests only the appliance
> 
> This feature doesn't test the CLI at all, but just the appliance.
> There are similar tests elsewhere already, where they belong.

Reply:
Also, it was the only test that used a `Transform` feature that's incompatible with Cucumber 3.